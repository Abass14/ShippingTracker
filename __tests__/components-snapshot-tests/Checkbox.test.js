import ShallowRenderer from 'react-shallow-renderer';
import Checkbox from '../../src/app/components/checkbox';

it('should render Checkbox correctly', () => {
    const renderer = new ShallowRenderer()
    const tree = renderer.render(
        <Checkbox isChecked onChecked={() => {}} />
    )
    expect(tree).toMatchSnapshot()
})