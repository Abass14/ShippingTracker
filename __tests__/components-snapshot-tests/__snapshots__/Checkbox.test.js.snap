// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Checkbox correctly 1`] = `
<TouchableOpacity
  onPress={[Function]}
>
  <CheckboxCheckedIcon
    size={20}
  />
</TouchableOpacity>
`;
